src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1053,6 +1053,38 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
             }
         }
 
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Not currently supported on UAP")]
+        [OuterLoop] // TODO: Issue #11345
```

Comment:
We're trying to remove the obsoleted TODO 11345.

Instead this should be "[Outerloop("Use external server")]"